src/Okta.Sdk/CreateUserWithImportedHashedPasswordOptions.cs

```diff
@@ -0,0 +1,40 @@
+﻿// <copyright file="CreateUserWithImportedHashedPasswordOptions.cs" company="Okta, Inc">
+// Copyright (c) 2014 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+namespace Okta.Sdk
+{
+    /// <summary>
+    /// Contains the options for creating a new <see cref="IUser">User</see> with imported hashed password.
+    /// Used with <see cref="IUsersClient.CreateUserAsync(CreateUserWithImportedHashedPasswordOptions, System.Threading.CancellationToken)"/>.
+    /// </summary>
+    /// <remarks>See <a href="https://developer.okta.com/docs/reference/api/users/#create-user-with-imported-hashed-password">Create User with Imported Hashed Password</a> in the documentation.</remarks>
+    public sealed class CreateUserWithImportedHashedPasswordOptions
```

Comment:
Is there precedent set for this naming convention?  Are other "Options" objects named using a similar convention?  Mostly want to be sure we're consistent, but this name "feels" like it could be better somehow.

Reply:
![image](https://user-images.githubusercontent.com/68990515/92118781-9a561e80-edff-11ea-971c-dc0adc86da2e.png)
I believe we consistent here